tests/transformers/tests/models/gpt2/test_modeling_gpt2.py

```diff
@@ -750,9 +750,9 @@ def test_gpt2_sample(self):
         )
         output_seq_strs = tokenizer.batch_decode(output_seq, skip_special_tokens=True)
         output_seq_tt_strs = tokenizer.batch_decode(output_seq_tt, skip_special_tokens=True)
-        EXPECTED_OUTPUT_STR = (
-            "Today is a nice day and if you don't know anything about the state of play during your holiday"
-        )
+
```

Comment:
Are these output difference also seen in OHF?

Reply:
Yeah, I shared these internally with you.